Caltech-IPAC/firefly
# DM-14157: hips/fits/aitoff switching

- buttons to move between hips/fits/aitoff
  - zoom switching now includes allsky in algorithm
  - auto check box to switch on zoom
  - zoom level switch to field of view
  - fixed: FOV when image zoomed level is really small
  - fixed: order of buttons
  - fixed: context toolbar going away when in different context modes
  - ButtonGroup option for RadioGroupInputFieldView

_to test:_

 - do catalog search, then switch between hips and fits
 - try the auto button and zoom
 - do a catalog search with m4, then m1
 - switch between hips, fits, aitoff
 - try auto button and zoom
 - try tooltip over field of view for both hips and fits
 - notice that there is only field of view, not zoom level
